client/src/app/components/identity/tree/IdentityTree.scss

```diff
@@ -0,0 +1,11 @@
+.identity-tree {
+  min-width: 200px;
+  max-height: 400px;
+  overflow-x: hidden;
+  overflow-y: auto;
+  scrollbar-width: thin;
```

Comment:
Can we match the scroll bar width to the same as other components, it's very hard to hit on a highdpi display

Reply:
I made it a little bit wider, But wider than that it will not look good because the tree is small (200px). Also on Firefox the scrollbar with the arrows doesn't look good at all but the thin one still wide enough I guess (and it removes the arrows which is good) So we have to sacrifice some usability for the looks here.